tracks/haskell.sh

```diff
@@ -1,10 +1,33 @@
+function _override_stack_resolver() {
+    if [[ -z "$XR_HASKELL_STACK_RESOLVER" ]]; then
+        return
+    fi
+
+    # See https://docs.haskellstack.org/en/stable/pantry/#snapshot-location for resolver syntax.
+    # For the purpose of running exercises it's resonable just to allow lts and nightly.
```

Comment:
```suggestion
    # See https://docs.haskellstack.org/en/stable/pantry/#snapshot-location for
    # the resolver syntax. For the purpose of running exercises, it's reasonable
    # just to allow LTS and nightly.
```

Reformatting and a small typo.